cli/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Integritee AG <hello@integritee.network>"]
 edition = "2021"
 
 [dependencies]
+array-bytes = { version = "6.0.0" }
```

Comment:
Hmm, not sure if I want to have this crate just for the `array_bytes::hex2bytes` function.

Reply:
I agree. But instead of removing it, one could also consider using it for all hex conversions. I've seen other hex functions that are defined within the worker.